### PR TITLE
Make sure strong elements return a bold font weight

### PR DESCRIPTION
**Pull request by frankie297**
## Why?
A side effect of the recent CSS reset introduced has resulted in <strong> tags not being styled with font-weight: bold. This means that Racing Tips, Sports articles, Letters etc have lost their “boldness”. This PR fixes that.

### CODE REVIEW

**Review by groakland — APPROVED** [review]
Lint nit aside, this looks good to go. Editions Production team can test this in code.preview and code.proofed and will verify all is in order

**Review by groakland — COMMENTED** [review]
Editions Production team have validated the fix and confirm that the text formatting is correct. This is the vital piece of the fix so would like this shipped ASAP

File: apps-rendering/src/renderer.ts [hunk 1]
```diff
@@ -223,7 +225,7 @@ const textElement =
 						key,
 						isEditions,
 					},
-					transform(text, format),
+					children,
```
Comment (marjisound) [inline apps-rendering/src/renderer.ts:228]: I'm trying to understand the reason the text of the anchor doesn't need to go through transform any more? If the content has `•` or `* * *` do we not need the styling of the Bullet & HorizontalRule?

Comment (frankie297) [reply apps-rendering/src/renderer.ts:228]: @JamieB-gu  can probably explain it better than me, but we can get rid of the transform as I understand it, transform is evaluated in the `#text` element further up: https://github.com/guardian/dotcom-rendering/blob/df0be1b184dc51315b7146bdfbf4a6da34682927/apps-rendering/src/renderer.ts#L215-L216 so any styles added there will be inherited via the children in anchor.

Comment (marjisound) [reply apps-rendering/src/renderer.ts:228]: Oh I see, the content of anchor would go through recursion and end up in `#text` case which is then applied to transform function. Makes sense, thanks for explaining it :)

**Review by marjisound — APPROVED** [review]

File: apps-rendering/src/renderer.ts (hunk 1 above)
Comment (marjisound) [reply apps-rendering/src/renderer.ts:228]: Oh I see, the content of anchor would go through recursion and end up in #text case which is then applied to transform function. Makes sense, thanks for explaining it :)

Comment (frankie297) [reply apps-rendering/src/renderer.ts:228]: Correct yes :)

**Pull request merged**